lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -263,7 +263,7 @@ def request(url, user, passwd, timeout, data=None, method=None):
     body = response.read()
 
     if body:
-        return json.loads(body)
+        return json.loads(body.decode('utf-8'))
```

Comment:
```suggestion
        return json.loads(to_text(body, errors='surrogate_or_strict'))
        
```

Reply:
[`to_text`](https://github.com/ansible/ansible/blob/9c1033422b53b7308f2ec7229222c8e7740f4566/lib/ansible/module_utils/_text.py#L169) is wrapper function.

Reply:
oh! okay.. understood! thank you.